updatesnap/updatesnapyaml.py

```diff
@@ -76,6 +83,8 @@ def main():
                         help='Version schema of snapping repository')
     parser.add_argument('--rock-version-schema', action='store', default='None',
                         help='Version schema of rock repository')
+    parser.add_argument('--yaml-path', action='store', default='None',
```

Comment:
I think that it would be better to use `None` instead of the string `"None"`. Like in `github-user` and `github-token`.

Also, I see that `version-schema` and `rock-version-schema` also use a string instead of the `None` native type... I think that it should be fixed in another PR.

Reply:
I will do it in new PR for version-schema and rock-version-schema.

Reply:
I think there is an issue when using None instead of 'None', as action.yml sends 'None' when an argument is not passed. We can't pass None as the default value in action.yml because it breaks.

Reply:
Maybe you should try a different way in the `action.yml` file. You can test if an environment variable is defined or not directly in the line, and add a piece of text if it is defined. For example:

    ${VERSION_SCHEMA:+--version-schema $VERSION_SCHEMA}

the `+:` part means "if the previous variable is defined, add what is after me". Thus, if VERSION_SCHEMA isn't defined, that will translate to no text, while if it is defined, it will translate to `--version-schema $VERSION_SCHEMA`